node/build_node_bridge.py

```diff
@@ -49,9 +53,23 @@ def main(args=None):
     if node_os_name.startswith('..\\'):
         node_os_name = node_os_name[3:]
 
+    cargo_target = options.cargo_target
+    if cargo_target is None:
+        print('ERROR: --cargo-target is required')
+        return 1
+    if cargo_target.startswith('..\\'):
+        cargo_target = cargo_target[3:]
+
+    node_arch = options.node_arch
+    if node_arch is None:
+        print('ERROR: --node_arch is required')
+        return 1
+    if node_arch.startswith('..\\'):
+        node_arch = node_arch[3:]
```

Comment:
@jack-signal, do you remember what this `..\` check is for? It doesn't seem like any of these would come up in arguments.

Reply:
It's certainly not supposed to. But it was showing up when invoked via `gyp` on Windows. I think the way `gyp` (or possibly Python) are parsing command line arguments on Windows is wrong, but fixing that was more than I wanted to deal with.

Reply:
Yeah I also had it show up on Windows, very weird indeed... I think it's Python's parsing of command line arguments on Windows though, rather than Gyp, IIRC